great_expectations/data_context/data_context.py

```diff
@@ -734,10 +772,12 @@ def run_validation_operator(
             run_id=run_id,
         )
 
-    def add_datasource(self, name, **kwargs):
+    def add_datasource(self, name, do_not_initialize=False, **kwargs):
```

Comment:
nit: given usage below, this is a weird double negative. shouldn't it be `initialize=True`

Reply:
Done.